src/hotspot/share/prims/jvmtiGetLoadedClasses.cpp

```diff
@@ -122,8 +118,9 @@ JvmtiGetLoadedClasses::getClassLoaderClasses(JvmtiEnv *env, jobject initiatingLo
   LoadedClassesClosure closure(env, true);
   {
     // To get a consistent list of classes we need MultiArray_lock to ensure
-    // array classes aren't created during this walk.
-    MutexLocker ma(MultiArray_lock);
+    // array classes aren't created during this walk. This walks through the
```

Comment:
Just curious but how can walking the set of classes trigger creation of array classes?

Reply:
It can't.  I'll reword that comment so it's clearer, with just a couple of words.

    // To get a consistent list of classes we need MultiArray_lock to ensure
    // array classes aren't created by another thread this walk. This walks through the
    // InstanceKlass::_array_klasses links.